saleor/product/migrations/0142_auto_20210201_1459.py

```diff
@@ -0,0 +1,160 @@
+# Generated by Django 3.1.5 on 2021-02-01 14:59
+
+from django.db import migrations, models
+import django.db.models.deletion
+import versatileimagefield.fields
+from ...product import ProductMediaTypes
+
+
+def migrate_images_content_to_media(apps, schema_editor):
+    ProductImage = apps.get_model("product", "ProductImage")
+    ProductMedia = apps.get_model("product", "ProductMedia")
+
+    VariantImage = apps.get_model("product", "VariantImage")
+    VariantMedia = apps.get_model("product", "VariantMedia")
+
+    for image in ProductImage.objects.all():
```

Comment:
```suggestion
    for image in ProductImage.objects.iterator():
```
We need to expect a lot of product images and we need to be sure that migration dont break.